libs/feature/record/src/lib/record-metadata/record-metadata.component.html

```diff
@@ -26,6 +26,27 @@
         </gn-ui-metadata-info>
       </div>
       <div>
+        <div
+          *ngIf="lightbox$ | async as lightbox"
+          data-cy="record-thumbnail"
```

Comment:
What is `data-cy`?  Is it something for cypress? If yes I cannot find it being used in the test file.

Reply:
It's indeed for cypress. I've directly added them in order to test but didn't provide test cases for logo and phone, but I created a test for address